docs/pages/introduction/why-not-expo.mdx

```diff
@@ -3,108 +3,110 @@ title: Limitations
 
-> If you'd like to continue getting the same development experience that you have with Expo Go, but also use custom native libraries or custom native code, you should check out the [Expo Dev Client](/development/introduction/)!
+</Collapsible>
 
-Expo Go comes with a pre-configured set of libraries, known as the Expo SDK. This makes development much faster, and essentially makes the mobile development experience much closer to the web development experience. However, additional native code cannot be added to Expo Go on the fly, so this means you are limited in terms of what dependencies you can add. This also applies to apps built with `expo build:android` and `expo build:ios`.
+<Collapsible summary="Native dependencies can be customized to target children under a particular age">
 
-<Collapsible summary="Not all iOS and Android APIs are available">
+Both [Apple](https://developer.apple.com/app-store/review/guidelines/#kids) and [Google](https://support.google.com/googleplay/android-developer/answer/9285070?hl=en) provide strict guidelines for any apps that specifically target children under the age of 13.
 
-Many device APIs are supported (check out the "SDK API Reference" in the sidebar), but **not all iOS and Android APIs are available yet**. We are constantly adding new APIs, so if we don't have something you need now, you can either use [EAS Build](/build/setup), the [bare workflow](managed-vs-bare#bare-workflow), or follow [our blog](https://blog.expo.dev) to see the release notes for our SDK updates. Feature prioritization isn't strictly based off of popular vote, but it certainly helps us to gauge what is important to users.
+Apps built with EAS Build [include only your app's explicit native dependencies](https://blog.expo.dev/expo-managed-workflow-in-2021-d1c9b68aa10). See [Development builds](/build/introduction/) for more information on how to use it.
 
 </Collapsible>
 
-<Collapsible summary={<span>If you need to keep your app size extremely lean, <InlineCode>expo build:[android|ios]</InlineCode> may not be the best choice</span>}>
+EAS Build is a new service and we plan to address many of the current limitations in time. See [EAS Build Limitations](/build-reference/limitations/) for more information on what are the current limitations.
 
-The size for an app built with `expo build` on iOS is approximately 20mb (download), and Android is about 15mb. This is because a bunch of APIs are included regardless of whether or not you are using them &mdash; this lets you publish updates that use new APIs, but comes at the cost of binary size. Some of the APIs that are included are tied to services that you may not be using, for example, the Facebook Mobile SDK is included to support Facebook Login and Facebook Ads, along with the Google Mobile SDK for similar reasons. [Read more about managing your app size here](https://expo.fyi/managed-app-size).
+## Expo Go and the classic `expo build` system
```

Comment:
I think we should separate out Expo Go and Classic Builds, since the former is sticking around. E.g., "Expo Go is and remains a great way to get started or quickly try something out, but you will need to graduate to dev builds once you use a library with native code that isn't built in."

I'm not even sure we need to mention classic builds in this doc. This is for devs considering Expo, and they shouldn't be jumping into classic builds.